website/docs/developer/research/research-bamboo-integration-tests.md

```diff
@@ -0,0 +1,129 @@
+A snippet of the script to deploy RDS cluster via terraform is shown below.
+```bash
+# Deploy drds-cluster-tf via terraform
+echo "Deploying rds-cluster-tf  module to $bamboo_DEPLOYMENT"
+terraform apply \
+  -auto-approve \
+  -lock=false \
+  -input=false \
+  -var-file="terraform.tfvars" \
+  -var "prefix=$bamboo_PREFIX" \
+  -var "region=$bamboo_AWS_DEFAULT_REGION" \
+  -var "subnets=[\"$bamboo_AWS_SUBNET_ID1\", \"$bamboo_AWS_SUBNET_ID2\"]" \
+  -var "db_admin_username=$bamboo_DB_ADMIN_USERNAME" \
+  -var "db_admin_password=$bamboo_DB_ADMIN_PASSWORD" \
+  -var "vpc_id=$bamboo_VPC_ID" \
+  -var "cluster_identifier=$bamboo_RDS_CLUSTER_ID" \
+  -var "deletion_protection=false"\
+  -var "provision_user_database=false"\
+  -var "engine_version=10.14"\
+  -var "permissions_boundary_arn=arn:aws:iam::$bamboo_AWS_ACCOUNT_ID:policy/$bamboo_ROLE_BOUNDARY"
+```
+Variables in terraform.tfvars can be overwritten by the `-var` field as seen above. 
+
+Some of the sensitive bamboo variables such as `AWS_ACCESS_KEY_ID`, `AWS_SECRET_ACCESS_KEY`, `RDS_USER_ACCESS_SECRET_ARN` have been encrypted using bamboo encryption service. While running the pipeline, those variables have to be replaced manually with the real values since bamboo does not automatically decrypt the values while running the pipeline. Make sure all sensitive variables are encrypted before pushing the changes to the repo. 
+
+The bamboo spec for running the two scripts under tasks is shown below.
+
+```yaml
+#job for deploying cumulus and orca
+Deploy RDS cluster:
+  key: DRC
+  other:
+    clean-working-dir: true
+    # Some plugin configurations are not supported by YAML Specs
+  docker:
+    image: amazonlinux:2
+    volumes:
+      ${bamboo.working.directory}: ${bamboo.working.directory}
+      ${bamboo.tmp.directory}: ${bamboo.tmp.directory}
+  tasks:
+  - checkout:
+      force-clean-build: 'true'
+      description: Checkout Default Repository
+
+  - script:
+      interpreter: SHELL
+      scripts:
+      - |-
+        #!/bin/bash
+        chmod +x bin/deployment-rds.sh
+        bin/deployment-rds.sh
+#job for deploying cumulus and orca
+Deploy cumulus and orca:
+  key: DCO
+  other:
+    clean-working-dir: true
+    # Some plugin configurations are not supported by YAML Specs
+  docker:
+    image: amazonlinux:2
+    volumes:
+      ${bamboo.working.directory}: ${bamboo.working.directory}
+      ${bamboo.tmp.directory}: ${bamboo.tmp.directory}
+  tasks:
+  - checkout:
+      force-clean-build: 'true'
+      description: Checkout Default Repository
+
+  - script:
+      interpreter: SHELL
+      scripts:
+      - |-
+        #!/bin/bash
+        chmod +x bin/deployment-cumulus-orca.sh
+        bin/deployment-cumulus-orca.sh
+```
+
+## Future directions
+
```

Comment:
I did not see any information on running in a Docker container, or the script snippets from Abdelhack on ways to better manage terraform variables needed for deployment using env files. I think I would add some basic information about that to help us understand as we move forward so things can be better/easier maintained.

Reply:
added links to GHRC dockerfile repo.I did create a card to look more into this. See [ORCA-393](https://bugs.earthdata.nasa.gov/browse/ORCA-393)